service/openapi.yaml

```diff
@@ -0,0 +1,246 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/evaluationFailure'
+        '401':
+          description: Unauthorized - You need credentials to access the API
+        '403':
+          description: Forbidden - You are not authorized to access the API
+        '429':
+          description: Rate limit reached on the Flag Management System
+          headers:
+            Retry-Later:
+              description: Indicates when to retry the request again
+              schema:
+                type: string
+                format: date-time
+                examples:
+                  - '2024-02-07T12:00:00Z'
+        '500':
+          description: Internal server error
+  /ofrep/v1/evaluate/flags/:
+    post:
+      description: OFREP bulk evaluation request
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/bulkEvaluationRequest'
+      responses:
+        '200':
+          description: OFREP successful evaluation response
+          headers:
+            ETag:
+              schema:
+                type: string
+              description: Entity tag used for cache validation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/bulkEvaluationSuccess'
+        '400':
+          description: Bad evaluation request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/bulkEvaluationFailure'
+        '401':
+          description: Unauthorized - You need credentials to access the API
+        '403':
+          description: Forbidden - You are not authorized to access the API
+        '429':
+          description: Rate limit reached on the Flag Management System
+          headers:
+            Retry-Later:
+              description: Indicates when to retry the request again
+              schema:
+                type: string
+                format: date-time
+                examples:
+                  - '2024-02-07T12:00:00Z'
+        '500':
+          description: Internal server error
+components:
+  securitySchemes:
+    BearerAuth:
+      description: (optional) Bearer Authorization to your flag management system.
+      type: http
+      scheme: bearer
+    ApiKeyAuth:
+      description: (optional) API Key to your flag management system.
+      type: apiKey
+      in: header
+      name: X-API-Key
+  schemas:
+    bulkEvaluationRequest:
+      description: Evaluate multiple flags in one request
+      properties:
+        flags:
+          $ref: '#/components/schemas/flags'
+        context:
+          $ref: '#/components/schemas/context'
+    bulkEvaluationSuccess:
+      description: Success response for the bulk evaluation request
+      type: object
+      properties:
+        flags:
+          type: array
+          items:
+            oneOf:
```

Comment:
I believe if you use `oneOf` it means all items in the list must have the same type. Isn't it possible to have a bulk where one flag is successful and another flag is a failure? I believe you need to use `anyOf` for that.

Reply:
@dyladan Yes, the bulk response should contain both success and failures. And I interpreted this rule to be an _array containing items that match oneOf the schemas from success or failuire_. I couldn't find a lot of resources on correct interpretation when oneOf is used within an array. But I came across this discussion [1] which suggests this is correct. 

[1] - https://community.smartbear.com/discussions/swaggerhub/the-use-of-oneof-or-anyof-in-arrays/257567